packages/experimental/Expander/windows/ReactNativeExpander/ExpanderView.cpp

```diff
@@ -0,0 +1,255 @@
+            else if (propertyName == "contentBorderBrush") {
+                this->SetResourceColor(propertyValue, L"ExpanderContentBorderBrush");
+            }
+            else if (propertyName == "chevronBackground") {
+                this->SetResourceColor(propertyValue, L"ExpanderChevronBackground");
+            }
+            else if (propertyName == "chevronForeground") {
+                this->SetResourceColor(propertyValue, L"ExpanderChevronForeground");
+            }
+            else if (propertyName == "chevronPointerOverBackground") {
+                this->SetResourceColor(propertyValue, L"ExpanderChevronPointerOverBackground");
+            }
+            else if (propertyName == "chevronPointerOverForeground") {
+                this->SetResourceColor(propertyValue, L"ExpanderChevronPointerOverForeground");
+            }
+            else if (propertyName == "chevronPressedBackground") {
+                this->SetResourceColor(propertyValue, L"ExpanderChevronPressedBackground");
+            }
+            else if (propertyName == "chevronPressedForeground") {
+                this->SetResourceColor(propertyValue, L"ExpanderChevronPressedForeground");
+            }
+            else if (propertyName == "chevronBorderThickness") {
+                if (!propertyValue.IsNull()) {
+                    auto resDict = expander.Resources();
+                    Windows::UI::Xaml::Thickness thickness = Windows::UI::Xaml::ThicknessHelper::FromUniformLength(propertyValue.AsDouble());
+                    resDict.Insert(winrt::box_value(L"ExpanderChevronBorderThickness"), winrt::box_value(thickness));
+                }
+            }
+            else if (propertyName == "chevronBorderBrush") {
+                this->SetResourceColor(propertyValue, L"ExpanderChevronBorderBrush");
+            }
+            else if (propertyName == "chevronBorderPointerOverBrush") {
+                this->SetResourceColor(propertyValue, L"ExpanderChevronBorderPointerOverBrush");
+            }
+            else if (propertyName == "chevronBorderPressedBrush") {
+                this->SetResourceColor(propertyValue, L"ExpanderChevronBorderPressedBrush");
+            }
+        }
+        m_updating = false;
+    }
+
+    void ExpanderView::SetResourceColor(Microsoft::ReactNative::JSValue const& value, hstring key) {
+        if (!value.IsNull()) {
+            auto expander = (this->Content()).try_as<Microsoft::UI::Xaml::Controls::Expander>();
+            auto color = value.To<winrt::Brush>();
+            auto resDict = expander.Resources();
+            resDict.Insert(winrt::box_value(key), color);
+        }
+    }
+
+    void ExpanderView::OnCollapsing(winrt::Windows::Foundation::IInspectable const& sender,
+                          Microsoft::UI::Xaml::Controls::ExpanderCollapsedEventArgs const& args) {
+        if (!m_updating) {
+            m_reactContext.DispatchEvent(
+                *this,
+                L"topCollapsing",
```

Comment:
Since you're firing `onChange` and  `onCollapsing` at the same time`, I think you can remove onCollapsing, as it doesn't seem to convey any extra information; same for onExpanding